src/layouts/Home.tsx

```diff
@@ -22,6 +22,26 @@ const Home = (): JSX.Element => {
       >
         Login with GitHub
       </Button>
+      <Text color="text.helper" fontSize="0.625rem" pt="1rem">
+        By clicking “Log in”, you are acknowledging and agreeing to Isomer’s{" "}
+        <Link
+          href="https://guide.isomer.gov.sg/terms-and-privacy/terms-of-use"
+          rel="noopener noreferrer"
+          target="_blank"
+          fontSize="0.625rem"
+        >
```

Comment:
actually i think chakra's `Link` has the `isExternal` property, which does this!

Reply:
Right, fixed in https://github.com/isomerpages/isomercms-frontend/pull/1003/commits/bd7a9f4fae5776ff8793241d67edcf99a76197a6